src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -63,13 +63,13 @@ private enum ResponseStatus
             long errorCode = authResult.Error.ErrorCode;
             string errorMessage;
 
-            logger.Info("[WamBroker] Processing WAM exception");
-            logger.Verbose(() => $"[WamBroker] TelemetryData: {authResult.TelemetryData}");
+            logger.Info("[RuntimeBroker] Processing WAM exception");
```

Comment:
Is it because the "old broker" had [WamBroker] ?

I'm not a fan of this - changing log statements like this will make it harder to find the statements.

Reply:
Yes the old broker has [WamBroker] and the new one has [[RuntimeBroker]](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs), just the WamAdapter.cs on the new one still has [WamBroker], while looking at logs this leads to a confusion